src/datagen/test_datagen.js

```diff
@@ -0,0 +1,177 @@
+/**
```

Comment:
Can we rename the file name to test_suitegen.js??

Reply:
Please read the file along with the folder name and see if it is understandable as a whole.
would src/generators/testdata.js, or src/generators/testsuites.js make sense?

Reply:
Addressed in [https://github.com/googleinterns/OAS-Adherence-Test-Suite/pull/4](url)